build/teamcity/cockroach/nightlies/microbenchmark_build_support.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+source "$dir/teamcity-support.sh"
+
+# Set up credentials
+google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+log_into_gcloud
+
+# Check if a given GCS path exists
+function check_gcs_path_exists() {
+  local path=$1
+  if gsutil ls "$path" > /dev/null 2>&1; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+# Build and copy binaries, for the given SHA, to GCS bucket
+function build_and_copy_binaries() {
```

Comment:
Maybe `upload` is a slightly better name than `copy`?

Reply:
Good point, I'll rename.